ocs_ci/cleanup/aws/cleanup.py

```diff
@@ -0,0 +1,56 @@
+import tempfile
+import argparse
+import logging
+import threading
+import os
+
+from ocs_ci.framework import config
+from ocs_ci.ocs.constants import CLEANUP_YAML, TEMPLATE_CLEANUP_DIR
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.utility import templating
+
+
+logger = logging.getLogger(__name__)
+
+
+def cleanup(cluster_name, cluster_id):
+    """
+    Cleanup existing cluster in AWS
+
+    Args:
+        cluster_name (str): Name of the cluster
+        cluster_id (str): Cluster id to cleanup
+
+    """
+    data = {'cluster_name': cluster_name, 'cluster_id': cluster_id}
+    template = templating.Templating(base_path=TEMPLATE_CLEANUP_DIR)
+    cleanup_template = template.render_template(CLEANUP_YAML, data)
+    cleanup_path = tempfile.mkdtemp(prefix='cleanup_')
+    with open(f"{cleanup_path}/metadata.json", "w") as temp:
```

Comment:
please use os.path.join for compose of path

Reply:
will fix both comment in next push, have to resolve conflicts as well.